src/components/checkbox/checkbox.spec.ts

```diff
@@ -294,14 +251,47 @@ describe('MdCheckbox', () => {
   describe('with provided aria-label ', () => {
     let checkboxDebugElement: DebugElement;
     let checkboxNativeElement: HTMLElement;
+    let inputElement: HTMLInputElement;
 
     it('should use the provided aria-label', async(() => {
       builder.createAsync(CheckboxWithAriaLabel).then(f => {
         fixture = f;
         checkboxDebugElement = fixture.debugElement.query(By.directive(MdCheckbox));
         checkboxNativeElement = checkboxDebugElement.nativeElement;
+        inputElement = <HTMLInputElement>checkboxNativeElement.querySelector('input');
+
+        fixture.detectChanges();
+        expect(inputElement.getAttribute('aria-label')).toBe('Super effective');
+      });
+    }));
+  });
+
+  describe('with provided aria-labelledby ', () => {
+    let checkboxDebugElement: DebugElement;
+    let checkboxNativeElement: HTMLElement;
+    let inputElement: HTMLInputElement;
+
+    it('should use the provided aria-labelledby', async(() => {
+      builder.createAsync(CheckboxWithAriaLabelledby).then(f => {
+        fixture = f;
+        checkboxDebugElement = fixture.debugElement.query(By.directive(MdCheckbox));
+        checkboxNativeElement = checkboxDebugElement.nativeElement;
+        inputElement = <HTMLInputElement>checkboxNativeElement.querySelector('input');
+
+        fixture.detectChanges();
+        expect(inputElement.getAttribute('aria-labelledby')).toBe('some-id');
+      });
+    }));
+
+    it('should not assign aria-labelledby if none is provided', async(() => {
+      builder.createAsync(SingleCheckbox).then(f => {
+        fixture = f;
+        checkboxDebugElement = fixture.debugElement.query(By.directive(MdCheckbox));
+        checkboxNativeElement = checkboxDebugElement.nativeElement;
+        inputElement = <HTMLInputElement>checkboxNativeElement.querySelector('input');
```

Comment:
These two tests duplicate some behavior. It would be good to just do it once in a `beforeEach`